web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.tsx

```diff
@@ -175,21 +171,14 @@ export const AnytimeActionTaxClearanceCertificateElement = (props: Props): React
                   <Heading level={1}>{props.anytimeAction.name}</Heading>
                 </div>
               </div>
-              {certificatePdfBlob ? (
-                <TaxClearanceDownload
-                  certificatePdfBlob={certificatePdfBlob}
-                  downloadFilename={`Tax Clearance Certificate - ${Date.now()}`}
-                />
-              ) : (
-                <TaxClearanceSteps
-                  steps={stateTaxClearanceCertificateSteps}
-                  currentStep={stepIndex}
-                  stepIndex={setStepIndex}
-                  saveTaxClearanceCertificateData={saveTaxClearanceCertificateData}
-                  setCertificatePdfBlob={setCertificatePdfBlob}
-                  setStepIndex={setStepIndex}
-                />
-              )}
+              <TaxClearanceSteps
+                steps={stateTaxClearanceCertificateSteps}
+                certificatePdfBlob={props.CMS_ONLY_certificatePdfBlob}
```

Comment:
We can still override for the CMS preview. I think this may also be a useful prop if we do endup storing the Tax Clearance Cert.